lib/app/client.js

```diff
@@ -98,7 +98,7 @@ function mapTransitions(Components, to, from) {
     if (from && from.matched.length && from.matched[0].components.default) {
       const from_transitions = componentTransitions(from.matched[0].components.default)
       Object.keys(from_transitions)
-        .filter((key) => from_transitions[key] && key.toLowerCase().indexOf('leave') !== -1)
+        .filter((key) => from_transitions[key] && key.toLowerCase().includes('leave'))
```

Comment:
Is `from_transitions[key]` necessary because some keys might have falsy values and we don't want those?

Anyway:
```js
const LEAVE_EX = /[Ll]eave/ // Could anything except the first letter be capital? 🤔
// ...
        .filter((key) => from_transitions[key] && LEAVE_EX.test(key))
```

Reply:
Won't add as it hurts readability/maintainability.